ui/src/app/cohort-search/search-group-list/search-group-list.component.tsx

```diff
@@ -85,14 +85,17 @@ const styles = reactStyles({
 });
 
 function mapMenuItem(item: CriteriaMenu) {
-  const { category, domainId, group, id, name, sortOrder, type } = item;
+  const { category, domainId, group, id, name, parentId, sortOrder, type } = item;
   return {
     category,
     children: null,
     domain: domainId,
     group,
     id,
     name,
+    selectedSurvey: domainId === Domain.SURVEY.toString() && parentId !== 0
+      ? name
+      : null,
```

Comment:
The api call to load a specific survey already exists for use in concept search. By adding the survey name as `selectedSurvey`, the `CriteriaSearch` component knows to display that specific survey instead of the entire tree.